Portal/Templates/AutoPropertyDiffable.stencil

```diff
@@ -5,10 +5,12 @@
     {% if type.accessLevel == "public" %}public {% endif %}enum Property {
 
         {% for instanceVariable in type.instanceVariables|publicGet %}
+        {% if not instanceVariable.annotations.ignoreInChangeSet %}
         {% if instanceVariable.type.based.AutoPropertyDiffable %}
-        case {{ instanceVariable.name }}([{{ instanceVariable.type.name }}.Property]{% if instanceVariable.isOptional %}?{% endif %})
+          case {{ instanceVariable.name }}([{{ instanceVariable.type.name }}.Property]{% if instanceVariable.isOptional %}?{% endif %})
         {% else %}
-        case {{ instanceVariable.name }}({{ instanceVariable.typeName }})
+          case {{ instanceVariable.name }}({{ instanceVariable.typeName }})
```

Comment:
Previous indentation was correct. If we add this space the generated code will wrongly indented.